FirebaseAppCheck/Sources/AppAttestProvider/DCAppAttestService+FIRAppAttestService.h

```diff
@@ -16,21 +16,22 @@
 
 #import <TargetConditionals.h>
 
+#import "FirebaseAppCheck/Sources/Public/FirebaseAppCheck/FIRAppCheckAvailability.h"
+
 // Currently DCAppAttestService is available on iOS only.
-#if TARGET_OS_IOS && !TARGET_OS_MACCATALYST  // Catalyst should be possible with Xcode 12.5+
+#if FIR_APP_ATTEST_SUPPORTED_TARGETS
 
 #import <DeviceCheck/DeviceCheck.h>
 
 #import "FirebaseAppCheck/Sources/AppAttestProvider/FIRAppAttestService.h"
 
 NS_ASSUME_NONNULL_BEGIN
 
-API_AVAILABLE(ios(14.0))
-API_UNAVAILABLE(macos, tvos, watchos)
+FIR_APP_ATTEST_PROVIDER_AVAILABILITY
```

Comment:
Does this still show appropriately in docs still?

Reply:
Hm, it doesn't, but I cannot find availability displayed anywhere in the docs for other APIs... Where should I check?

Reply:
The documentation and generated Swift API is not different from ones when `API_AVAILABLE` is added directly.